README.md

```diff
@@ -5,18 +5,17 @@
 [![PyPi](https://img.shields.io/pypi/pyversions/meeshkan)](https://pypi.org/project/meeshkan/)
 [![License](https://img.shields.io/pypi/l/meeshkan)](LICENSE)
 
-Meeshkan is a tool for mocking HTTP APIs for use in sandboxes as well as for automated and exploratory testing. It uses a combination of API definitions, recorded traffic and code in order to make crafting mocks as enjoyable as possible.
+Meeshkan is a tool that mocks HTTP APIs for use in sandboxes as well as for automated and exploratory testing. It uses a combination of API definitions, recorded traffic and code in order to make crafting mocks as enjoyable as possible.
 
 Meeshkan requires Python 3.6 or later.
 
 ## Table of Contents
 
 1. [Installation](#installation)
-1. [Python API](#python-api)
-1. [Builder](#builder)
-1. [Mocking](#mocking)
-1. [Recording](#recording)
-1. [Converting](#converting)
+1. [Hello world](#hello-world)
+1. [Collect](#collect)
+1. [Build](#build)
+1. [Mock](#mock)
```

Comment:
I love the idea of mimicking the workflow in the README structure! 

My only note is that these headers should be able to stand on their own (as should each section generally). So we should make these more specific.